ansible/README.md

```diff
@@ -0,0 +1,53 @@
+# Ansible playbook to deploy PyGrid components (for Debian based systems)
+
+## Install ansible
+
+Before you try run any ansible script, you need install ansible. You can find more
+information about installation to your linux distribution [here](http://docs.ansible.com/ansible/intro_installation.html).
+
+If you use Debian based distribution, you can install using the commands below:
```

Comment:
The `README` already mentions Debian-based distributions. I'd rather remove this mention here and add instructions of how to also install `ansible` as a pip package under a `virtualenv`.